salt/modules/selinux.py

```diff
@@ -456,7 +456,7 @@ def fcontext_get_policy(name, filetype=None, sel_type=None, sel_user=None, sel_l
     '''
     if filetype:
         _validate_filetype(filetype)
-    re_spacer = '[ ]{2,}'
+    re_spacer = '[ ]+'
```

Comment:
this should probably just be changed to `'\s+'`? to grab all possible whitespace instead of just spaces?

Reply:
Potentially. The output is always just space characters though and I'd prefer the parsing we do to reflect the format as specifically as possible.